workers/ohsome_quality_analyst/indicators/mapping_saturation/indicator.py

```diff
@@ -194,3 +152,29 @@ def create_figure(self) -> None:
         self.result.svg = img_data.getvalue()  # this is svg data
         logging.debug("Successful SVG figure creation")
         plt.close("all")
+
+    def check_corner_cases(self) -> bool:
```

Comment:
maybe not so important, but I think `edge_case` might be the better word? 

* https://en.wikipedia.org/wiki/Edge_case
* https://en.wikipedia.org/wiki/Corner_case

Reply:
I changed the word to edge case.